editor.planx.uk/src/@planx/components/ContactInput/model.ts

```diff
@@ -23,6 +23,22 @@ export const userDataSchema: SchemaOf<Contact> = object({
       "Enter an email address in the correct format, like name@example.com"
     )
     .required("Enter an email address"),
+}).test({
+  name: "Test Test is not used for applications",
+  test: ({ firstName, lastName }, context) => {
+    const isValid =
+      [firstName, lastName]
+        .map((x) => String(x).toLowerCase().trim())
+        .join("|") !== "test|test";
+
+    if (isValid) return true;
+
+    return context.createError({
```

Comment:
Due to some Formik / Yup integration issues, we need to manually throw an error here instead of just returning false and including a `message` in the object. A good number of issues on the Formik repo related to this, solution was found here - https://github.com/jaredpalmer/formik/issues/2146#issuecomment-720639988